avocado/utils/lv_utils.py

```diff
@@ -58,7 +58,8 @@ def get_diskspace(disk):
 
 
 def vg_ramdisk(disk, vg_name, ramdisk_vg_size,
-               ramdisk_basedir, ramdisk_sparse_filename):
+               ramdisk_basedir, ramdisk_sparse_filename,
+               use_tmpfs=True):
```

Comment:
OK, tested with https://github.com/avocado-framework-tests/avocado-misc-tests/blob/master/io/disk/lvsetup.py, with the following changes:

```diff
diff --git a/io/disk/lvsetup.py b/io/disk/lvsetup.py
index 2f8d124..3c476ec 100755
--- a/io/disk/lvsetup.py
+++ b/io/disk/lvsetup.py
@@ -74,6 +74,7 @@ class Lvsetup(Test):
             'ramdisk_basedir', default=os.path.join(self.workdir, 'ramdisk'))
         self.ramdisk_sparse_filename = self.params.get(
             'ramdisk_sparse_filename', default='virtual_hdd')
+        self.use_tmpfs = self.params.get('use_tmpfs', default=False)
 
         if lv_utils.vg_check(vg_name):
             self.cancel('Volume group %s already exists' % vg_name)
@@ -100,7 +101,8 @@ class Lvsetup(Test):
         self.ramdisks.append(lv_utils.vg_ramdisk(self.disk, self.vg_name,
                                                  self.ramdisk_vg_size,
                                                  self.ramdisk_basedir,
-                                                 self.ramdisk_sparse_filename))
+                                                 self.ramdisk_sparse_filename,
+                                                 self.use_tmpfs))
         lv_utils.lv_create(self.vg_name, self.lv_name, self.lv_size)
         lv_utils.lv_mount(self.vg_name, self.lv_name, self.mount_loc,
                           create_filesystem=self.fs_name)
```

Reply:
I guess these changes are added to the commit or one of the batch branches?